### PR TITLE
A cosmetic change for 09f989d4. Show [brackets] around directories names in the status bar if nothing is selected.

### PR DESCRIPTION
**Pull request by corvus1**
A bit of obsessive mimicking of Total Commander. The discussion is in the end of #207 
Please consider.

### CODE REVIEW

**Comment by alexx2000** [comment]
Wrong code, it should be same as [here](https://github.com/doublecmd/doublecmd/blob/master/src/ufilefunctions.pas#L217).

**Comment by corvus1** [comment]
Oh. I didn't realize that. :-) Well, I'm unfamiliar with the codebase, but I didn't introduce any new dependencies, not a single `uses`.

I'll try to rework it, but I'll have to take a good long look into it. :-)

**Comment by Skif-off** [comment]
See `<FolderPrefix>` and `<FolderPostfix>` [here](https://doublecmd.github.io/doc/en/configxml.html).

Simething like
```pas
      if AFile.IsNameValid then
      begin
        if AFile.IsDirectory then
          lblInfo.Caption := gFolderPrefix + AFile.Name + gFolderPostfix
        else
          lblInfo.Caption := AFile.Name
      else if not AInfo then begin
        inherited UpdateInfoPanel;
      end;
```
without `F`?

**Comment by corvus1** [comment]
@Skif-off 

Yes, I figured it out, quicker than I originally thought. The prefix and postfix come from the settings, and AFile has IsDirectory, so there's not reason to check it separately.

There's, however, one more problem I'm not sure how to solve, maybe you'll help me a little. The way I did it, it doesn't add brackets on DC start. When you start the program, the directory your cursor was on the last time shows in the status bar with no brackets, until you move the cursor, then it updates and shows the brackets.

So apparently that function doesn't get called on DC start, which might or might not indicate a bigger problem, actually. But to locate that I'd really have to familiarize myself with the way the whole thing works.

**Comment by corvus1** [comment]
I pushed the correct implementation, and the other problem I was talking about is somehow... gone (?), fixed itself (?). IDK, I'll take it. Problem fixing themselves is nice. :-)

File: src/fileviews/ucolumnsfileview.pas [hunk 1]
```diff
@@ -743,15 +743,20 @@ procedure TColumnsFileView.UpdateFooterDetails(AInfo: Boolean);
...
     AFile:= CloneActiveFile;
     if Assigned(AFile) then
-    try
-      if AFile.IsNameValid then
-        lblInfo.Caption := AFile.Name
-      else if not AInfo then begin
-        inherited UpdateInfoPanel;
+      try
+        if AFile.IsNameValid then begin
+          if AFile.IsDirectory then
```
Comment (alexx2000) [inline src/fileviews/ucolumnsfileview.pas:748]: It should be `if gDirBrackets and (AFile.IsDirectory or AFile.IsLinkToDirectory) then` as I mentioned above. It should has same display name as name on the file panel.

Comment (corvus1) [reply src/fileviews/ucolumnsfileview.pas]: Yeah, sorry for missing that. Fixed.

**Comment by corvus1** [comment]
Thank you, everyone! My first contribution. :1st_place_medal:

**Pull request merged**